presto-cli/src/main/java/io/prestosql/cli/OutputHandler.java

```diff
@@ -13,85 +13,93 @@
  */
 package io.prestosql.cli;
 
-import io.airlift.units.Duration;
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
I think you might have a merge conflict here.  The license is already above the `package` statement.

Reply:
Thanks Dain. Fixed it.